### PR TITLE
Copy DS changes

### PR DESCRIPTION
**Pull request by iranzo**
# Description

Changes from DS PR's:

https://github.com/openshift/openshift-docs/pull/27656
https://github.com/openshift/openshift-docs/pull/27654
https://github.com/openshift/openshift-docs/pull/27653
https://github.com/openshift/openshift-docs/pull/27652
https://github.com/openshift/openshift-docs/pull/27650

### CODE REVIEW

File: documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
 do
   sudo virsh destroy $i;
   sudo virsh undefine $i;
-  sudo virsh vol-delete $i --pool default;
+  sudo virsh vol-delete $i --pool $i;
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc:60]: Unsure about this, as the volume is in $i but the pool should be default right?

File: documentation/ipi-install/modules/ipi-install-creating-an-rhcos-images-cache.adoc [hunk 2]
```diff
@@ -26,6 +26,7 @@ Use the following steps to install a container that contains the images.
 [source,terminal]
 ----
 [kni@provisioner ~]$ sudo firewall-cmd --add-port=8080/tcp --zone=public --permanent
+[kni@provisioner ~]$ sudo firewall-cmd --reload
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-creating-an-rhcos-images-cache.adoc:29]: ACK from me

File: documentation/ipi-install/modules/ipi-install-extracting-the-openshift-installer.adoc [hunk 3]
```diff
@@ -38,4 +38,5 @@ After retrieving the installer, the next step is to extract it.
 ----
 [kni@provisioner ~]$ sudo cp oc /usr/local/bin
 [kni@provisioner ~]$ oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
+[kni@provisioner ~]$ sudo cp openshift-baremetal-install /usr/local/bin
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-extracting-the-openshift-installer.adoc:41]: ACK from me

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc [hunk 4]
```diff
@@ -23,8 +23,10 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
 do
   sudo virsh destroy $i;
   sudo virsh undefine $i;
-  sudo virsh vol-delete $i --pool default;
+  sudo virsh vol-delete $i --pool $i;
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:26]: Same as above, we're removing $i, but the pool is default, isn't it?

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc [hunk 5]
```diff
@@ -23,8 +23,10 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
...
   sudo virsh vol-delete $i.ign --pool default;
+  sudo virsh pool-destroy $i;
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:28]: ok from me

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc [hunk 6]
```diff
@@ -23,8 +23,10 @@ for i in $(sudo virsh list | tail -n +3 | grep bootstrap | awk {'print $2'});
...
+  sudo virsh pool-undefine $i;
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:29]: same here

File: documentation/ipi-install/modules/ipi-install-creating-an-rhcos-images-cache.adoc (hunk 2 above)
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-creating-an-rhcos-images-cache.adoc:29]: ACK from me

File: documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc (hunk 1 above)
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc:60]: Unsure about this, as the volume is in $i but the pool should be default right?

**Review by iranzo — COMMENTED** [review]
@rlopez133 please check my comments on the changes based on the DS modifications

File: documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc (hunk 1 above)
Comment (rlopez133) [reply documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc:60]: Your right this should say default as $i is only capturing the volume

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc (hunk 4 above)
Comment (rlopez133) [reply documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:26]: Default is the name, technically can be named anything. For simplicity, I think we leave default as the name.

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc (hunk 5 above)
Comment (rlopez133) [reply documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:28]: It won't be able to destroy a pool of $i as its not the actual name. Going back to your above concern.

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc (hunk 6 above)
Comment (rlopez133) [reply documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:29]: yep

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc (hunk 5 above)
Comment (iranzo) [reply documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:28]: I meant to destroy it, but yes, using 'default'

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc [hunk 7]
```diff
@@ -25,6 +25,8 @@ do
   sudo virsh undefine $i;
   sudo virsh vol-delete $i --pool default;
   sudo virsh vol-delete $i.ign --pool default;
+  sudo virsh pool-destroy default;
+  sudo virsh pool-undefine default;
 done
 ----
```
Comment (iranzo) [inline documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:48]: [root@worker-0 ~]# sudo virsh pool-list 
setlocale: No such file or directory
 Name                   State    Autostart
--------------------------------------------
 default                active   yes
 kni4-25rhx-bootstrap   active   yes
 kni4-27qxs-bootstrap   active   yes
 kni4-2rrwq-bootstrap   active   yes
 kni4-2tqbn-bootstrap   active   yes
 kni4-4rsbl-bootstrap   active   yes
 kni4-54hr9-bootstrap   active   yes
 kni4-6mz49-bootstrap   active   yes
 kni4-76dds-bootstrap   active   yes
 kni4-76gvw-bootstrap   active   yes
 kni4-7bbq9-bootstrap   active   yes
 kni4-7xfvk-bootstrap   active   yes
 kni4-8fg2k-bootstrap   active   yes
 kni4-b6l7v-bootstrap   active   yes
 kni4-b9m89-bootstrap   active   yes
 kni4-c4jf6-bootstrap   active   yes
 kni4-d4pb8-bootstrap   active   yes
 kni4-grl68-bootstrap   active   yes
 kni4-hscxh-bootstrap   active   yes
 kni4-jfvhd-bootstrap   active   yes
 kni4-jhthd-bootstrap   active   yes
 kni4-jnv6z-bootstrap   active   yes
 kni4-jrmgk-bootstrap   active   yes
 kni4-kgzvx-bootstrap   active   yes
 kni4-ks4b7-bootstrap   active   yes
 kni4-l2vr7-bootstrap   active   yes
 kni4-ljrkk-bootstrap   active   yes
 kni4-nfzv6-bootstrap   active   yes
 kni4-nq9f8-bootstrap   active   yes
 kni4-q6r9t-bootstrap   active   yes
 kni4-rl8zr-bootstrap   active   yes
 kni4-rqw52-bootstrap   active   yes
 kni4-td6f6-bootstrap   active   yes
 kni4-v4n5r-bootstrap   active   yes
 kni4-v6b7t-bootstrap   active   yes
 kni4-vdmn2-bootstrap   active   yes
 kni4-vfx58-bootstrap   active   yes
 kni4-vz976-bootstrap   active   yes
 kni4-wjwvs-bootstrap   active   yes
 kni4-wr9tg-bootstrap   active   yes
 kni4-wtbwv-bootstrap   active   yes
 kni4-xx5fs-bootstrap   active   yes
 kni4-zcn58-bootstrap   active   yes

Comment (iranzo) [reply documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:48]: We need a conditional then here...

File: documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc (hunk 1 above)
Comment (iranzo) [reply documentation/ipi-install/modules/ipi-install-configuring-the-install-config-file.adoc:61]: Applied conditional to >= 4.6 and to < 4.6

File: documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc (hunk 4 above)
Comment (iranzo) [reply documentation/ipi-install/modules/ipi-install-troubleshooting-cleaning-up-previous-installations.adoc:27]: Same conditionals here (check output from kni4)

**Comment by iranzo** [comment]
4.3 version: https://deploy-preview-597--baremetal-deploy.netlify.app/4.3/deployment#configuring-the-install-config-file_ipi-install-configuration-files

image


4.6 version: https://deploy-preview-597--baremetal-deploy.netlify.app/4.6/deployment#configuring-the-install-config-file_ipi-install-configuration-files

image

**Pull request merged**